tests/testthat/test_PlainTextParam.R

```diff
@@ -12,42 +12,80 @@ test_that("storeResults,loadResults,PlainTextParam,MsBackendMzR works", {
     ## Loading data again
     b2 <- loadResults(object = MsBackendMzR(), param)
     expect_true(inherits(b2, "MsBackendMzR"))
-    b <- dropNaSpectraVariables(b) #the function does this to be robust, is it a problem ? i should mention it in the doc
+    b <- dropNaSpectraVariables(b)
     expect_equal(b@spectraData, b2@spectraData)
-    expect_equal(peaksVariables(b), peaksVariables(b2))  # true even without forcing the slot
+    expect_equal(peaksVariables(b), peaksVariables(b2))
+    expect_equal(mz(b[1:20]), mz(b2[1:20]))
+
+    ## Check the spectraPath parameter.
```

Comment:
this is to simulate moving a stored results to a different computer and the usage of the `spectraPath` parameter.